src/core/middlewares/regtoken/handler.go

```diff
@@ -0,0 +1,55 @@
+package regtoken
+
+import (
+	"context"
+	"github.com/goharbor/harbor/src/common/utils/log"
+	"github.com/goharbor/harbor/src/core/config"
+	"github.com/goharbor/harbor/src/core/middlewares/util"
+	pkg_token "github.com/goharbor/harbor/src/pkg/token"
+	"github.com/goharbor/harbor/src/pkg/token/claim"
+	"net/http"
+	"strings"
+)
+
+type regTokenHandler struct {
```

Comment:
Please add comment to explain what this handler does.

Reply:
sure, added.